goon/browserassets/js/browserOutput.js

```diff
@@ -126,7 +126,7 @@ function highlightTerms(el) {
 							break;
 						}
 					}
-					newText += newWord ? newWord : words[w];
+					newText += (newWord ? newWord : words[w]).replace("<", "&lt;");
```

Comment:
You can do text entities, but the worst thing you can do with those is reverse text